appveyor.yml

```diff
@@ -4,69 +4,132 @@ clone_depth: 1
 skip_commits:
     message: /(iiod|IIOD):.*/
 
+environment:
+    # Tell msys2 to add mingw64 to the path
+    MSYSTEM: MINGW64
+    # Tell msys2 to inherit the current directory when starting the shell
+    CHERE_INVOKING: 1
+    matrix:
+        - COMPILER: mingw
+        - COMPILER: msvc
+
+configuration:
+    - Release
+
 install:
+    - if "%COMPILER%"=="mingw" (set GENERATOR=Unix Makefiles)
+    - if "%COMPILER%"=="msvc" (set GENERATOR=Visual Studio 12)
     - echo "Checking out sub-modules..."
     - git submodule update --init
+    - cd ..
```

Comment:
Is this cd necessary? It is only followed by a echo and then another cd to an absolute location